eventhubs/client/azure-eventhubs/src/main/java/com/azure/core/amqp/CBSNode.java

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.amqp;
+
+import reactor.core.publisher.Mono;
+
+import java.io.Closeable;
+import java.time.Duration;
+
+/**
+ * Claims-based security (CBS) node that authorizes connections with AMQP services.
+ *
+ * @see <a href="https://www.oasis-open.org/committees/download.php/62097/amqp-cbs-v1.0-wd05.doc">
+ * AMPQ Claims-based Security v1.0</a>
+ */
+public interface CBSNode extends Closeable {
```

Comment:
Why did you go with Closeable rather than AutoCloseable?

Reply:
I wanted to avoid the entire try-with-resources statement that the Service team had pointed out was probably a bad design because it could lead to developers continuously creating new links and sessions rather than keeping the same instance as long as possible. But I also wanted to make it clear that they had to dispose of these AmqpConnections, Sessions, eventually.

Reply:
That's fine with me. We should be sure to document this in JavaDoc when you get to that stage.